pycheribuild/projects/cross/nginx.py

```diff
@@ -113,3 +119,10 @@ class BuildFettNginx(BuildNginx):
     project_name = "fett-nginx"
     repository = GitRepository("https://github.com/CTSRD-CHERI/nginx.git",
         default_branch="fett")
+    dependencies = ["fett-openssl"]
+
+    def configure(self):
+        openssl_dir = str(BuildFettOpenSSL.get_instance(self)._installPrefix)
+        self.configureEnvironment["NGX_OPENSSL_fett_path"] = str(BuildFettOpenSSL.get_instance(self).destdir) + openssl_dir
```

Comment:
Does `self.configureEnvironment["NGX_OPENSSL_fett_path"] = str(BuildFettOpenSSL.getInstallDir(self))` work?

Reply:
Ah never mind you need the install prefix for the rpath anyway.